packages/proposals

```diff
@@ -0,0 +1,53 @@
+1 PKG_PRIORITY
+
+None of the current scripts use the PKG_PRIORITY variable.
+All but three packages set PKG_PRIORITY="optional". busybox, dbus and systemd
+set PKG_PRIORITY="required".
+The effect of PKG_PRIORITY is unknown.
+
+Proposal:
+Deprecate the PKG_PRIORITY variable
+
+
+2 PKG_SITE, PKG_LONGDESC, PKG_SHORTDESC
+
+None of the current scripts use the PKG_SITE variable.
+create_addon is the only script that uses the PKG_LONGDESC and PKG_SHORTDESC
+variables.
+In other words, the PKG_LONGDESC and PKG_SHORTDESC are only used for
+addons. For other packages, the PKG_SITE, PKG_LONGDESC and PKG_SHORTDESC are
+merely informative and redundant.
+
+Proposal:
+Deprecate PKG_LONGDESC and/or PKG_SHORTDESC for normal packages
+
+
+3 PKG_AUTORECONF
+
+More than 330 packages set PKG_AUTORECONF="no", whereas about 230 packages set
+PKG_AUTORECONF="yes".
+build is the only script that uses the PKG_AUTORECONF variable.
+
+Proposal: 
+Modify build to set PKG_AUTORECONF="no" if PKG_AUTORECONF is not set in
+in package.mk
+
+
+4 PKG_LICENCE
+
+Values of PKG_LICENSE are inconsistent among themselves and with the licenses in
+/licenses.
+
+Proposal:
+Update PKG_LICENSE and /licenses
+
+
+5 PKG_ADDON_REPOVERSION
+
+The PKG_ADDON_REPOVERSION is set individually in the package.mk file of each
+addon. The PKG_ADDON_REPOVERSION however pertains to the release of LibreELEC.
+create_addon is the only script that uses the PKG_ADDON_REPOVERSION.
+
+Proposal:
+Define PKG_ADDON_REPOVERSION centrally with LibreELEC, modify create_addon
+accordingly, deprecate PKG_ADDON_REPOVERSION in package.mk.
```

Comment:
as you dont have a repo setup like https://github.com/OpenELEC/OpenELEC.tv/blob/master/packages/addons/official/repository/repository.unofficial.addon.pro/sources/addon.xml#L8 you dont need PKG_ADDON_REPOVERSION anymore. you rebuild everything after new LE release, and backport changes to old repos anywya, so drop it.